### ISSUE TITLE
[Sync] remoteStorage support

### ISSUE THREAD

**Issue by ahmubashshir** [sienori/Tab-Session-Manager#1068]
Please add support for @remotestorage sync as a lot of people (probably not as many as google drive users though...) may look for a self-hosted solution for session sync[^*].

<sub> Or you could always add support for other self-hosted alternatives out there like nextcloud/... too</sub>

[^*]: I'd go for a self-hosted solution before turning to google :3

**Comment by Zachiah** [comment]
Yes, please! I don't want google using all of my open tabs in an ad campaign 😉. Having an option to use Nextcloud, remotestorage, WebDAV, or others would be awesome!

**Comment by SergeyDjam** [comment]
And Syncthing, please
